src/Standards/Squiz/Tests/WhiteSpace/MemberVarSpacingUnitTest.1.inc.fixed

```diff
@@ -375,3 +375,24 @@ class NoPreambleMultilineDeclaration {
     static
     int $prop = 1;
 }
+
+class MultipleBlankLinesInPreAmble {
+
+    /**
+     * Docblock.
+     */
+    #[MyAttribute]
+    #[MyOtherAttribute]
+    public $prop;
+}
+
+final class BlankLinesBetweenVsAttributesWithoutCommentIssueSquiz3594
+{
+
+    // phpcs can fix this but not the next one
+    #[SingleAttribute]
+    public $property1;
+
+    #[SingleAttribute]
+    public $property2;
+}
```

Comment:
What hasn't been fixed for the second property here? Does the comment need to be updated?

Reply:
See: https://github.com/squizlabs/PHP_CodeSniffer/issues/3594

I've updated the comment now. Is this better ?